### ISSUE TITLE
training on test set in note_03

### ISSUE THREAD

**Issue by DrAnaximandre** [chiphuyen/tf-stanford-tutorials#8]
Hi,

in `note_03.pdf,` p13, in the testing model part, the code states:
`_, loss_batch, logits_batch = sess.run([optimizer, loss, logits], feed_dict={X: X_batch, Y:Y_batch})`.

Does this call to `optimizer` trains the model on the test set and has an impact on the next epoch? 

Thanks for your great course material.

**Comment by zengyu714** [comment]
I guess it should have effects on the update of `weights/biases`.  So it might be a slip of the pen. @DrAnaximandre

**Comment by chiphuyen** [comment]
fixed. thanks
